src/com/flashfla/net/Multiplayer.as

```diff
@@ -1009,7 +1066,10 @@ package com.flashfla.net
             }
 
             if (user == currentUser)
-                sendCurrentUserRoomVariables(room);
+            {
+                var JOINING:Boolean = true;
```

Comment:
Caps aren't used for function-scoped constants, let alone variables. Also this would be `const`, not `var`. Also this doesn't change anywhere, so just pass `true` to `sendCurrentUserRoomVariables`.

Reply:
Sending booleans in to functions is dumb, so I tend to pass a constant bool in so it's super easily readable. But I'll change it up anyway.